package.json

```diff
@@ -11,7 +11,13 @@
     "build": "rollup --config",
     "commitmsg": "validate-commit-msg",
     "destroy": "rm -rf node_modules package-lock.json dist",
-    "deploy": "npm run destroy && npm install && npm run build && aws s3 cp ./dist/webloader.js  s3://evolv-execution-plan-binaries/asset-manager/releases/latest/webloader.js --acl public-read && aws s3 cp ./dist/webloader.min.js  s3://evolv-execution-plan-binaries/asset-manager/releases/latest/webloader.min.js --acl public-read"
+    "prerelease": "npm ci && npm run build && npm run deploy:version:s3 && npm run deploy:next:s3 && npm run deploy:next:npm",
+    "release": "npm ci && npm run build && npm run deploy:version:s3 && npm run deploy:latest:s3 && npm run deploy:latest:npm",
```

Comment:
`pre<script>` and `post<script>` have special meanings to `NPM`. When running script `name`, NPM will run any script prefixed with `pre` before the `name` script and any script prefixed with `post` after.

```
{
  "scripts" : {
    "preexample": "echo Before",
    "example": "echo Example",
    "postexample": "echo After"
  }
}

$ npm run example
Before
Example
After
``` 

Also it's not conventional to place `npm ci` or `npm install` inside of a script. `npm ci` and `npm install` are usually treated as one time commands, run at the beginning of a build process.

Lastly, I often use a dev dependency named [npm-run-all](https://www.npmjs.com/package/npm-run-all) to keep my scripts organized and manageable. With it, you can use glob-matching to select scripts to run and then choose to run those in sequence or in parallel

Example _package.json_
```
{
  "scripts" : {
    "deploy": "npm-run-all -p deploy:*",
    "deploy:s3": "...",
    "deploy:npm": "..."
  },
  "devDependencies": {
    "npm-run-all": "4.1.5"
  }
}
```